src/main/resources/db_scripts/liquibase/liquibase-init.sql

```diff
@@ -0,0 +1,60 @@
+
+INSERT INTO databasechangelog (ID, AUTHOR, FILENAME, DATEEXECUTED, ORDEREXECUTED, MD5SUM, DESCRIPTION, COMMENTS, EXECTYPE, CONTEXTS, LABELS, LIQUIBASE, DEPLOYMENT_ID) VALUES ('v0.1.0.add-started-finished', 'kevin.wallimann@absa.africa', 'v0.1.0.add-started-finished', NOW(), 2, '8:f70938e6d526d71fd3874adfd52e79c1', 'sqlFile', '', 'EXECUTED', 'default', NULL, '3.10.2', '1336601767');
+
+INSERT INTO databasechangelog (ID, AUTHOR, FILENAME, DATEEXECUTED, ORDEREXECUTED, MD5SUM, DESCRIPTION, COMMENTS, EXECTYPE, CONTEXTS, LABELS, LIQUIBASE, DEPLOYMENT_ID) VALUES ('v0.1.3.add-key-value-pairs', 'kevin.wallimann@absa.africa', 'v0.1.3.add-key-value-pairs', NOW(), 3, '8:e8cebc04cbd5b57fb5fc7f7986ef1341', 'sqlFile', '', 'EXECUTED', 'default', NULL, '3.10.2', '1336601767');
+
+INSERT INTO databasechangelog (ID, AUTHOR, FILENAME, DATEEXECUTED, ORDEREXECUTED, MD5SUM, DESCRIPTION, COMMENTS, EXECTYPE, CONTEXTS, LABELS, LIQUIBASE, DEPLOYMENT_ID) VALUES ('v0.2.0.add-workflow-history', 'kevin.wallimann@absa.africa', 'v0.2.0.add-workflow-history', NOW(), 4, '8:a786a8311fbe19c5db31f7fbbf5c3dd9', 'sqlFile', '', 'EXECUTED', 'default', NULL, '3.10.2', '1336601767');
+
+INSERT INTO databasechangelog (ID, AUTHOR, FILENAME, DATEEXECUTED, ORDEREXECUTED, MD5SUM, DESCRIPTION, COMMENTS, EXECTYPE, CONTEXTS, LABELS, LIQUIBASE, DEPLOYMENT_ID) VALUES ('v0.2.0.add-workflow-id-to-dag-run-view', 'kevin.wallimann@absa.africa', 'v0.2.0.add-workflow-id-to-dag-run-view', NOW(), 5, '8:7ddae7eced01d161e731e75d7e504a86', 'sqlFile', '', 'EXECUTED', 'default', NULL, '3.10.2', '1336601767');
+
+INSERT INTO databasechangelog (ID, AUTHOR, FILENAME, DATEEXECUTED, ORDEREXECUTED, MD5SUM, DESCRIPTION, COMMENTS, EXECTYPE, CONTEXTS, LABELS, LIQUIBASE, DEPLOYMENT_ID) VALUES ('v0.2.0.add-job-template', 'kevin.wallimann@absa.africa', 'v0.2.0.add-job-template', NOW(), 6, '8:ac942942e1af37b9f3b51640941bfa0f', 'sqlFile', '', 'EXECUTED', 'default', NULL, '3.10.2', '1336601767');
+
+INSERT INTO databasechangelog (ID, AUTHOR, FILENAME, DATEEXECUTED, ORDEREXECUTED, MD5SUM, DESCRIPTION, COMMENTS, EXECTYPE, CONTEXTS, LABELS, LIQUIBASE, DEPLOYMENT_ID) VALUES ('v0.2.0.delete-job-type', 'kevin.wallimann@absa.africa', 'v0.2.0.delete-job-type', NOW(), 7, '8:dc26d32dc134c39cbc9c3e990d9fb99c', 'sqlFile', '', 'EXECUTED', 'default', NULL, '3.10.2', '1336601767');
+
+INSERT INTO databasechangelog (ID, AUTHOR, FILENAME, DATEEXECUTED, ORDEREXECUTED, MD5SUM, DESCRIPTION, COMMENTS, EXECTYPE, CONTEXTS, LABELS, LIQUIBASE, DEPLOYMENT_ID) VALUES ('v0.2.0.add-triggered-by', 'kevin.wallimann@absa.africa', 'v0.2.0.add-triggered-by', NOW(), 8, '8:20947f500206c9f3dcbced94d9079582', 'sqlFile', '', 'EXECUTED', 'default', NULL, '3.10.2', '1336601767');
```

Comment:
AUTHOR should be HyperdriveDevTeam@absa.africa ? 
I don't mind leaving it this way

Reply:
Hmm. The important thing is to track who made the change. To that end, the personal email is better. However, you can find out the author using the filename and git blame as well. So we can also use the group email address so as not to write down our personal emails into the DB

Reply:
I changed it